cockroachcloud/migrations-page.md

```diff
@@ -139,64 +139,92 @@ If the credentials are valid, they will be added to the [**Credentials** table](
 
 ## Review the schema
 
+Use the **Summary Report** and **Statements** list to [update the schema](#update-the-schema) and finalize it for migration.
+
 ### Summary Report
 
-The **Summary Report** displays the results of the schema analysis:
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+The **Summary Report** displays the results of the schema analysis and provides bulk actions you can apply to [update the schema](#update-the-schema).
+
+The banner at the top of the page displays:
 
 <ul>
 <li>The number of <b>Statements Total</b> in the uploaded <code>.sql</code> file that were analyzed.</li>
-<li>The number of <b>Errors</b> in SQL statements that are blocking <a href="#finalize-the-schema">finalization</a>. Errors are further categorized and counted on the <a href="#statement-status"><b>Statement Status</b></a> graph.</li>
+<li>The number of <b>Errors</b> in SQL statements that are blocking <a href="#migrate-the-schema">schema migration</a>. Errors are further categorized and counted in the <a href="#summary-report"><b>Summary Report</b></a>.</li>
 <li>The number of <b>Incidental Errors</b> in SQL statements that are caused by errors in other SQL statements.</li>
 
 <section class="filter-content" markdown="1" data-scope="mysql oracle mssql">
 <li>The number of <b>Incompatible Statements</b> that could not be converted because they have no equivalent syntax on CockroachDB.</li>
-<li>The number of <b>Compatibility Notes</b> regarding differences in SQL syntax. Although these statements do not block finalization, you should [update](#update-the-schema) them before finalization.</li>
+<li>The number of <b>Compatibility Notes</b> on differences in SQL syntax. Although these statements do not block schema migration, you should [update](#update-the-schema) them before migrating the schema.</li>
 </section>
 
 <li>The number of <b>Suggestions</b> regarding <a href="../{{version_prefix}}migration-overview.html#schema-design-best-practices">CockroachDB best practices</a>.</li>
 </ul>
 
-To [update the schema](#update-the-schema), click **View Statements** or the **Statements** tab to open the [**Statements** list](#statements-list). Errors and suggestions are displayed for each statement.
+To apply bulk actions to statements, refer to the tables in the Summary Report:
```

Comment:
Suggestion: This content in the second half of the Summary Report section could be under a subheading called "Update the schema" or "Apply fixes" (with Required Fixes + Suggested Fixes nested under it). Right now, it feels like the actions a user can take are lumped in with a description of the Summary Report and it's hard to visually distinguish between the two

(You can take or leave this suggestion)

Reply:
Actually I realized the banner content applies to both Summary Report & Statements list, so I was able to move it out and clarify this section. Thanks!